content/en/docs/topics/kubernetes_distros.md

```diff
@@ -16,14 +16,17 @@ alphabetically) if desired.
 
 ## AKS
 
-Helm works with [Azure Kubernetes
-Service](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm).
+Helm works with [Azure Kubernetes Service](https://docs.microsoft.com/en-us/azure/aks/kubernetes-helm).
```

Comment:
in #743 we started to word wrap at 80 characters for easier readability. The prior change was intentional here. Can you please revert this?

Reply:
Done.